server/SwipeDecision.py

```diff
@@ -15,10 +16,58 @@ def getPotentialMatchList(currentUserId):
             cursor.execute(PotentialMatchQuery, userID)
             potentialListId = [i[0] for i in cursor.fetchall()]
 
+            newPotentialMatchList = []
```

Comment:
I think you should put the finding the potential list based on the location into a new method that returns the new potential mist list